package.json

```diff
@@ -195,7 +195,7 @@
     "pod-install": "react-native setup-ios-permissions && cd ios && pod install",
     "pod-install-m1": "react-native setup-ios-permissions && cd ios && arch -x86_64 pod install",
     "postinstall": "patch-package && ./scripts/postinstall.sh",
-    "preinstall": "./scripts/preinstall.sh && npx solidarity",
```

Comment:
i have removed it until we fix this command @larkox

Reply:
Where is it broken?

Reply:
The first time we install the development environment the `android` executable doesn't exist. It somehow gets created while building the app for the first time (or whatever other action for the first time on Android Studio, I haven't look deep into it). So for it to pass the first time, we have to disable it for a moment.

That being said, instead of this heavy approach, I would only remove the "android executable exist" rule from the solidarity scripts (in `.solidarity`).

Reply:
again, solidarity should be kept

Reply:
reverted this change